static/app/views/performance/landing/widgets/components/accordion.tsx

```diff
@@ -0,0 +1,91 @@
+import {ReactNode} from 'react';
+import styled from '@emotion/styled';
+
+import DropdownButton from 'sentry/components/dropdownButton';
+import space from 'sentry/styles/space';
+
+type Props = {
+  content: ReactNode;
+  items: (() => ReactNode)[];
+  selectedIndex: number;
+  setSelectedIndex: (index: number) => void;
+};
+
+export default function Accordion({
+  content,
+  items,
+  selectedIndex,
+  setSelectedIndex,
+}: Props) {
+  return (
+    <AccordionContainer>
+      {items.map((item, index) => (
+        <AccordionItem
+          isSelected={index === selectedIndex}
+          currentIndex={index}
+          key={index}
+          content={content}
+          setSelectedIndex={setSelectedIndex}
+        >
+          {item()}
+        </AccordionItem>
+      ))}
+    </AccordionContainer>
+  );
+}
+
+function AccordionItem({
+  isSelected,
+  currentIndex: index,
+  children,
+  setSelectedIndex,
+  content,
+}: {
+  children: ReactNode;
+  content: ReactNode;
+  currentIndex: number;
+  isSelected: boolean;
+  setSelectedIndex: (index: number) => void;
+}) {
```

Comment:
Nit, but I think the combination of `content` and `children` is a little confusing. They both sound like the main content! Since you're choosing an API where `items` are the accordion item "headers", and the `content` is the body of each accordion item, I think it'd be good to name them that way! I think `items` and `content` makes sense for `Accordion`. Here, maybe `content` and `headerContent`, or something else descriptive?

Reply:
I agree, it's pretty confusing
I renamed `items` to `headers` to reflect that it's an array prop